src/test/java/dev/merge/client/BasicJavaTest.java

```diff
@@ -10,13 +10,15 @@
 import dev.merge.client.shared.MergePaginatedResponse;
 import junit.framework.TestCase;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Disabled;
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
 
 public class BasicJavaTest extends TestCase {
 
     @Test
+    @Disabled
```

Comment:
ignore tests since they need api tokens we don't want in code, and gradle publish now required in github ci